frontend/src/components/Utils/time.js

```diff
@@ -10,54 +10,30 @@ import { firestore } from 'firebase';
  * @returns {string} Time formatted into a string like '10:19 AM'.
  */
 export function timestampToTimeFormatted(msTimestamp, timezone = 'America/New_York') {
-  const date = new Date(msTimestamp);
-  const formatOptions = { 
-    hour: 'numeric', 
-    minute: '2-digit', 
-    timeZone: timezone
-  };
-  return date.toLocaleTimeString('en-US', formatOptions);
+  return moment.tz(parseFloat(msTimestamp), timezone).format('LT');
```

Comment:
I think it might help make it clearer to comment what the string in the format function means? Since right now "LT" doesn't mean anything to me.